pytensor/tensor/nlinalg.py

```diff
@@ -266,7 +266,9 @@ def __str__(self):
         return "SLogDet"
 
 
-slogdet = Blockwise(SLogDet())
+def slogdet(x):
```

Comment:
Can you add some docstrings also? Can probably reuse the initial stuff from the numpy docs, just reference TensorVariables instead
 
```suggestion
def slogdet(x: TensorLike) -> tuple[TensorVariable, TensorVariable]:
```